### PR TITLE
add findSimilarIncidentsByText script

### PR DESCRIPTION
**Pull request by erezh31**

### CODE REVIEW

File: Scripts/script-FindSimilarIncidentsByText.yml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+  INCIDENT_TEXT_FIELD = 'incident_text_for_tfidf'
+  MIN_TEXT_LENGTH = 50
+  MAX_CANDIDATES_IN_LIST = 10
+
+  from sklearn.feature_extraction.text import TfidfVectorizer
+
+  def parse_datetime(datetime_str):
+      datetime_str = datetime_str[:len('yyyy-mm-ddThh:mm:ss')]
+      return datetime.strptime(datetime_str, "%Y-%m-%dT%H:%M:%S")
```
Comment (meirwah) [inline Scripts/script-FindSimilarIncidentsByText.yml]: check for backward compatibility

File: Scripts/script-FindSimilarIncidentsByText.yml [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+      demisto.results('No similar incidents has been found')
+type: python
+tags:
+- ml
+comment: |
+  Find similar incidents by text comparison - the algorithm based on TF-IDF method.
+  To read more about this method: https://en.wikipedia.org/wiki/Tf%E2%80%93idf
+enabled: true
+args:
+- name: textFields
```
Comment (meirwah) [inline Scripts/script-FindSimilarIncidentsByText.yml]: default should be name + details

File: Scripts/script-FindSimilarIncidentsByText.yml (hunk 1 above)
Comment (erezh31) [reply Scripts/script-FindSimilarIncidentsByText.yml]: checked

File: Scripts/script-FindSimilarIncidentsByText.yml (hunk 2 above)
Comment (erezh31) [reply Scripts/script-FindSimilarIncidentsByText.yml]: V

**Comment by erezh31** [comment]
@meirwah it's failed on other tests, can merge

**Pull request merged**